scripts/jest/preprocessor.js

```diff
@@ -58,4 +63,6 @@ module.exports = {
     }
     return src;
   },
+
+  getCacheKey: createCacheKeyFunction([__filename]),
```

Comment:
this should also include babel's package.json and the path to the two babel plugins from above.